contracts/FeralFileAirdropV1.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "./Authorizable.sol";
+
+contract FeralFileAirdropV1 is ERC1155, Authorizable {
+    enum Type {
+        Fungible,
+        NonFungible
+    }
```

Comment:
I'm not sure if keeping the strict Fungible/NonFungible rule is good. For example what if we have an activation that contains multi edition and multi-unique edition

Reply:
I dont think it should happen. If it happened, we need to create v2, we cannot assume that.